Sources/Compound/Colors/CompoundColors.swift

```diff
@@ -125,3 +146,19 @@ private extension UITraitCollection {
     /// Whether or not the trait collection contains a `userInterfaceStyle` of `.light`.
     var isLight: Bool { userInterfaceStyle == .light }
 }
+
+public struct AvatarColor: Equatable {
+    let background: Color
+    let foreground: Color
+}
+
+private extension String {
+    /// Calculates a numeric hash same as Element Web
+    /// See original function here https://github.com/matrix-org/matrix-react-sdk/blob/321dd49db4fbe360fc2ff109ac117305c955b061/src/utils/FormattingUtils.js#L47
+    var hashCode: Int {
+        let charCodeSum = self.reduce(0) { sum, char in
+            return sum + Int(char.unicodeScalars.first?.value ?? 0)
```

Comment:
I think the return can be implicit here?
```suggestion
            sum + Int(char.unicodeScalars.first?.value ?? 0)
```
*Realises the repo needs SwiftLint and SwiftFormat*.